osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -85,10 +95,86 @@ public override double Calculate(Dictionary<string, double> categoryRatings = nu
 
             return totalValue;
         }
-
+public double[] Main()
+    {
+        try
+        {   // Open the text file using a stream reader.
+            string tempstor = Path.GetTempPath();
+            string jdapath = tempstor + "jda.txt";
+            using (StreamReader sr = new StreamReader(jdapath))
+            {
+            // Read the stream to a string, and write the string to the console.
+                double[] errorval = {1,2,3};
+                double[] JumpDistanceArray = new double[32767];
```

Comment:
this function always returns a double array with length 32767, you should just use linq, i.e. `a.Split(',').Select(r => Convert.ToDouble(r)).ToArray();`

Reply:
I could implement your code if I was proficient enough in C# but for now I'll use whatever makes the code readable and works simply.

Reply:
Fixed in #8